corehq/apps/reports/tests/test_tasks.py

```diff
@@ -1,27 +1,18 @@
-from datetime import datetime
+
```

Comment:
Is this an empty line on top of the file?

Reply:
Nice catch 4fff2cb